scripts/test/jest.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+jest
```

Comment:
I don't necessarily mind this set up if you really like it. Is there any value over 'consistency' here in setting `npm run jest` to point to `scripty` which effectively just runs `jest`? For some of the more complex tasks above, yea I can see the value. Am wondering if this will deter future collaboration or add confusion for folks (me) not familiar with this way of running the show.